### ISSUE TITLE
Missing releases on author page

### ISSUE THREAD

**Issue by trapd00r** [metacpan/metacpan-web#2136]
I'm sure it's me doing something wrong, but I just can't figure out what.
There's a few missing entries on my author page: https://metacpan.org/author/WOLDRICH
For example https://metacpan.org/release/WOLDRICH/App-makedist-0.012

The "Something's missing?" link suggested it was a missing

`package App::makedist;`
statement, so I added it but to no avail.

The dist in question doesn't show up when you search on metacpan.org, but you can find it via google.

What am I doing wrong?

**Comment by trapd00r** [comment]
I know for a fact that https://metacpan.org/release/WOLDRICH/App-xkcd-0.071 showed on my metacpan authors page a while back. What has changed?

**Comment by oalders** [comment]
The browse link on this release is messed up as well: https://metacpan.org/source/WOLDRICH/App-makedist-0.012

**Comment by oalders** [comment]
The API is not seeing your dist in `06perms.txt`: https://metacpan.org/permission/distribution/App-makedist

**Comment by trapd00r** [comment]
Am I to blame?

**Comment by oalders** [comment]
Here's what we see for you in `06perms.txt`:

https://metacpan.org/permission/author/WOLDRICH

Did you give away maint on any of the modules that are missing?  We're not good at historical permissions data, so something that previously appeared *could* disappear if you give away maint.  

For `App-makedist-0.012` what did the indexing email from PAUSE say?

**Comment by trapd00r** [comment]
No, I haven't given away any maintenance.
Also, I haven't received any mail from pause...

**Comment by trapd00r** [comment]
Yeah, I had an outdated mail adress linked in my pause account, so that's that. I'm trying to re-index some file from within my pause account now - cause there's a lot of dists missing from the metacpan listing it seems...

**Comment by trapd00r** [comment]
"You are not registred in any modulelist"
^ not sure if that's relevant?

**Comment by oalders** [comment]
Can you contact the PAUSE admins and see what they have to say about this?

**Comment by trapd00r** [comment]
So it seems a few things worked together in this issue:

* Back in the day we never used any *META.{yml,json} files, and a message from PAUSE suggested this was the culprit. I added them, still to no avail.
* Next, PAUSE suggested I should add a provides field like so:

```
  META_ADD                     => {
    provides => {
      'App::makedist' => {
        file => 'bin/makedist',
      },
    },
  },
```
since this dist doesn't have any .pm file. Still to no avail, though.

* (WARNING: META): I generated my dist App::makedist using my app makedist and I discovered that I implemented a bug: if your distribution doesn't unpack into a single directory, it won't be indexed no matter what. 

With all of these changes, I get a:
```

Status: Successfully indexed
============================

     module : App::makedist
     version: undef
     in file: bin/makedist
     status : indexed
```
**However**, it still doesn't show up in the metacpan. :D
